content/doc/developer/development-environment/index.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: developerchapter
+wip: true
+summary: Preparing your development environment for Jenkins core and plugin development
+references:
+- url: https://wiki.jenkins-ci.org/display/JENKINS/IntelliJ+IDEA+plugin+for+Stapler
+  title: IntelliJ IDEA plugin for Stapler
+- url: https://wiki.jenkins-ci.org/display/JENKINS/NetBeans+plugin+for+Stapler
+  title: NetBeans plugin for Stapler
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Setting+up+Eclipse+to+build+Jenkins
+  title: Setting up Eclipse to build Jenkins
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+plugin+development+with+Eclipse
+  title: Jenkins plugin development with Eclipse
+- url: https://wiki.jenkins-ci.org/display/JENKINS/Eclipse+alternative+build+setup
+  title: Eclipse alternative build setup
```

Comment:
That's what I thought as well. FWIW the "set up your IDE and how to do stuff" is one of my highest priority items.